modules/core/server/controllers/core.dbmodel.controller.js

```diff
@@ -1053,18 +1059,17 @@ _.extend (DBModel.prototype, {
 				.select(fields)
 				.exec(function(error, data) {
 					if (!error) {
-						// console.log('search.completed, get total count');
 						self.model.find(q).and(and).count(function(e,c) {
 							if (e) {
 								console.log('search.count.error = ' + JSON.stringify(e));
 								self.complete(reject, 'search');
 							} else {
-								// console.log('search.count.completed. total = ', c);
+								if (debug) console.log('search.count.completed. total = ', c);
 								resolve({data: data, count: c});
 							}
 						});
 					} else {
-						// console.log('search.error = ' + JSON.stringify(error));
+						console.log('search.error = ' + JSON.stringify(error));
```

Comment:
core.helpers.controller.js is a general server-side utility.  We should be putting common helper code in there.  Make a helpers.logger() function call in there, so that we can migrate over time.